x/commercioid/querier.go

```diff
@@ -75,7 +80,7 @@ func queryGetConnections(ctx sdk.Context, path []string, keeper Keeper) (res []b
 
 	bz, err2 := codec.MarshalJSONIndent(keeper.cdc, connectionsResult)
 	if err2 != nil {
-		panic("Could not marshal result to JSON")
+		return nil, sdk.ErrUnknownRequest(fmt.Sprintf("Could not marshal result to JSON"))
```

Comment:
If we do not have any error variables I think we can remove the usage of `fmt.Sprintf` and leave the plain string.